src/index.html

```diff
@@ -12,6 +12,41 @@
             <div id="container" class="container"></div>
         </section>
     </main>
+    <script>
+        function enableDragSort(listClass) {
```

Comment:
- [ ] **OPTIONAL**
Kindly remember that it's better to separate your logic code to a new file. As described in the project requirements, you should create a new file for the drag and drop functionalities. 
_I'm sorry this point didn't show up earlier 😊🙏🏻_